docker/linux/Dockerfile

```diff
@@ -1,84 +1,112 @@
+# Use Ubuntu 22.04 as the base image
 FROM ubuntu:22.04
 
+# Create the necessary directory
+RUN mkdir -p /ansys_inc && \
+    echo "Directory created."
+
+# Copy distribution files to /tmp
+COPY distributions/*.tgz /tmp/
+
+# Update package list, install tar, and extract the .tgz files in one RUN command
+RUN apt-get update && \
+    apt-get install -y tar && \
+    cd /tmp && \
+    for file in *.tgz; do tar -xzf "$file" -C /ansys_inc; done && \
+    ls -l /ansys_inc && \
+    rm -rf /tmp/* && \
+    echo "Temporary files removed."
+
+RUN apt-get update && apt-get install -y \
+    libbrotli1 \
+    libbsd0 \
+    libexpat1 \
+    libfontconfig1 \
+    libfreetype6 \
+    libglib2.0-0 \
+    libice6 \
+    libmd0 \
+    libpng16-16 \
+    libsm6 \
+    libx11-6 \
+    libx11-xcb1 \
+    libxau6 \
+    libxcb-icccm4 \
+    libxcb-image0 \
+    libxcb-keysyms1 \
+    libxcb-randr0 \
+    libxcb-render-util0 \
+    libxcb-render0 \
+    libxcb-shape0 \
+    libxcb-shm0 \
+    libxcb-sync1 \
+    libxcb-util1 \
+    libxcb-xfixes0 \
+    libxcb-xinerama0 \
+    libxcb-xkb1 \
+    libxcb1 \
+    libxdmcp6 \
+    libxext6 \
+    libxkbcommon-x11-0 \
+    libxkbcommon0
+
 ARG VERSION
-ARG MAPDL_PATH
 ARG USERNAME=mapdl
 ARG USER_UID=1000
 ARG USER_GID=$USER_UID
+ARG LICENSE_SERVER
 
-# Storing version in a env var
+
+# install the product
+RUN /ansys_inc/INSTALL -silent -install_dir /ansys_inc -licserverinfo $LICENSE_SERVER
+
+
+# Storing version in an env var
 ENV ANSYS_VERSION=$VERSION
 
 # Location env var
-# This var needs to be updated when version changes.
 ENV AWP_ROOT222=/ansys_inc
 
-
 # LABELS
 LABEL description="MAPDL on Ubuntu"
 LABEL email="pyansys.core@ansys.com"
 
 # OCI LABELS
 LABEL org.opencontainers.image.documentation="https://mapdl.docs.pyansys.com"
 
-# Update packages before packing and
-# installing dependencies
-RUN apt-get update && \
-    apt-get install -y \
-      sudo \
-      git \
-      python3.10-venv \
-      openssh-client \
-      libgl1 \
-      libglu1 \
-      libxm4 \
-      libxi6
-
-# libxp.so missing. This is a workaround
-#https://bugs.launchpad.net/ubuntu/+source/libxp/+bug/1517884
+# Update packages before installing dependencies
+
+# Workaround for missing libxp.so
 RUN apt install -y software-properties-common \
     && add-apt-repository -y ppa:zeehio/libxp \
     && apt-get update \
     && apt-get install -y libxp6
 
-
 # Create the user
 RUN groupadd --gid $USER_GID $USERNAME \
     && useradd --uid $USER_UID --gid $USER_GID -m $USERNAME
 
-
-# Copying files
-RUN mkdir -p /ansys_inc
-COPY $MAPDL_PATH /ansys_inc
-
-# Cleaning
-RUN rm -rf .dockerignore \
-    && rm -rf Dockerfile \
-    && rm -rf *.log
-
+# Cleaning up unnecessary files
+RUN rm -rf /tmp/*
```

Comment:
Remove only the right files in the `tmp/` dir. There may be files in this directory used by other running processes.